app/models/user.rb

```diff
@@ -0,0 +1,27 @@
+class User < ApplicationRecord
+  belongs_to :role
```

Comment:
ko cần tách bằng, dùng field role là dc
field này là enum in rails hì

Reply:
Dạ ok c